react/features/filmstrip/components/web/Filmstrip.js

```diff
@@ -243,6 +324,19 @@ class Filmstrip extends PureComponent <Props> {
     constructor(props: Props) {
         super(props);
 
+        this._filmstripRef = React.createRef();
+        this._dragRef = React.createRef();
```

Comment:
Isn't there more `React`y way of doing the whole drag thing? I mean without references.